docker/pipelines_runner.sh

```diff
@@ -51,6 +51,10 @@ function parse_args {
 }
 
 function main {
+  if [[ $1 == /opt/gcp_variant_transforms/bin/* ]]; then
+    $@
```

Comment:
I think you want to `exec` it instead so the return status is correct

Reply:
Done.